src/templates/Practice5.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+
+
+interface Data {
+  id: number;
+  todayDate: string;
+  taskName: string;
+  isChecked: boolean;
+}
+
+const Practice5: React.FC = () => {
+  const [todos, setTodos] = useState<Data[]>([]);
+  const [task, setTask] = useState<string>('');
+  const [allSelected, areAllSelected] = useState<boolean>(false);
```

Comment:
To Follow our [React guidelines ](https://pj100.esa.io/posts/16722#%E3%82%A2%E3%83%B3%E3%83%81%E3%83%91%E3%82%BF%E3%83%BC%E3%83%B3%E3%81%AE%E6%94%B9%E5%96%84)

```suggestion
  const [isAllSelected, setIsAllSelected] = useState<boolean>(false);
```

Reply:
Made the changes.